examples/rust-examples/mips64/scheduler/cooperative/src/main.rs

```diff
@@ -0,0 +1,63 @@
+#![no_std]
+#![no_main]
+
+use core::sync::atomic::{AtomicU32, Ordering};
+use martos::{
+    init_system,
+    task_manager::{TaskManager, TaskManagerTrait},
+};
+
+#[panic_handler]
+fn panic(_info: &core::panic::PanicInfo) -> ! {
+    loop {}
+}
+
+static COUNTER: AtomicU32 = AtomicU32::new(1);
+
+/// Setup function for the main task.
+fn main_task_setup_fn() {}
+
+/// Loop function for the main task.
+fn main_task_loop_fn() {
+    let count = COUNTER.fetch_add(1, Ordering::Relaxed);
+    if count == 25 {
+        TaskManager::add_task(
+            inner_task_setup_fn,
+            inner_task_loop_fn,
+            inner_task_stop_condition_fn,
+        );
+    }
+}
+
+/// Stop condition for the main task.
+fn main_task_stop_condition_fn() -> bool {
+    false
+}
```

Comment:
It's better to add condition for stopping main task to see, that inner task starts execution for all cooperative schedule examples